sources/web/datalab/polymer/components/datalab-toolbar/datalab-toolbar.html

```diff
@@ -24,6 +24,9 @@
 <dom-module id="datalab-toolbar">
   <template>
     <style include="datalab-shared-styles">
+      [hidden] {
```

Comment:
I think we should move this (and its other occurrences) into the `shared-styles` element.